src/requests.rs

```diff
@@ -512,10 +515,6 @@ pub fn outgoing_request_to_js_value(
         OutgoingRequests::RoomMessage(request) => {
             JsValue::from(RoomMessageRequest::try_from((request_id, request))?)
         }
-
-        OutgoingRequests::KeysBackup(request) => {
-            JsValue::from(KeysBackupRequest::try_from((request_id, request))?)
-        }
```

Comment:
`OutgoingRequests` was deleted in latest version of `matrix-sdk-crypto` (https://github.com/matrix-org/matrix-rust-sdk/commit/3ccd2e9f8fb2763ea8be46fbebf79d8807d2851f)